File: perl/src/tests/t_logging.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$logging = Atrshmlog::get_logging();

print "logging : $logging : \n";

$oldlogging = Atrshmlog::set_logging_process_off();

print "logging : $oldlogging : \n";

$logging = Atrshmlog::get_logging();

print "logging : $logging : \n";

$oldlogging = Atrshmlog::set_logging_process_on();

print "logging : $oldlogging : \n";

$logging = Atrshmlog::get_logging();

print "logging : $logging : \n";

$area = Atrshmlog::get_area();

$old = Atrshmlog::set_area_ich_habe_fertig($area, 1);

print "logging : $old : \n";

$logging = Atrshmlog::get_logging();

print "logging : $logging : \n";

$old = Atrshmlog::set_area_ich_habe_fertig($area, 0);

print "logging : $old : \n";

$logging = Atrshmlog::get_logging();

print "logging : $logging : \n";

$oldlogging = Atrshmlog::set_logging_process_off_final();

print "logging : $oldlogging : \n";

$logging = Atrshmlog::get_logging();

print "logging : $logging : \n";

print " \n";

exit (0);

# end of main

File: python/src/tests/t_logging.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

logging = atrshmlog.get_logging()

print('logging : ' + str(logging) + ' : ')

oldlogging = atrshmlog.set_logging_process_off()

print('logging : ' + str(oldlogging) + ' : ')

logging = atrshmlog.get_logging()

print('logging : ' + str(logging) + ' : ')

oldlogging = atrshmlog.set_logging_process_on()

print('logging : ' + str(oldlogging) + ' : ')

logging = atrshmlog.get_logging()

print('logging : ' + str(logging) + ' : ')

area = atrshmlog.get_area()

old = atrshmlog.set_area_ich_habe_fertig(area, 1)

print('logging : ' + str(old) + ' : ')

logging = atrshmlog.get_logging()

print('logging : ' + str(logging) + ' : ')

old = atrshmlog.set_area_ich_habe_fertig(area, 0)

print('logging : ' + str(old) + ' : ')

logging = atrshmlog.get_logging()

print('logging : ' + str(logging) + ' : ')

oldlogging = atrshmlog.set_logging_process_off_final()

print('logging : ' + str(oldlogging) + ' : ')

logging = atrshmlog.get_logging()

print('logging : ' + str(logging) + ' : ')


print (' ')

exit(0)

# end of test

